python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -485,9 +488,8 @@ def __call__(self, measure_input, tmp_dir, **kwargs):
         """
         tic = time.time()
         try:
-            filename = os.path.join(
-                tmp_dir, "tmp_func_%0x.%s" % (getrandbits(64), self.build_func.output_format)
-            )
+            output_format = "stackvm" if self.build_func is None else self.build_func.output_format
```

Comment:
This seems to to be a really specialized default, can you introduce a stackvm build func instead?

Reply:
I set the stackvm `build_func` to `None` because I found [this](https://github.com/apache/tvm/blob/83baec30d4418ac1470c696097913aafbfc8ad48/python/tvm/runtime/module.py#L323-L330) in existing code. In that segment of code, stackvm is calling `Module.save()` to dump stackvm binary, while the function further called a TVM runtime API `ModuleSaveToFile`. I can certainly make it a `build_func` for it but I'm a bit uncertain whether it's suitable to place it among those `contrib` ones.

Reply:
i think making build_func is more consistent than putting the logic inside.